### PR TITLE
BaSP-TG-55: Create shared buttons

### PR DESCRIPTION
**Pull request by LucianoFranchino**
- Created confirm/submit button
- Created cancel button
- Apply this btns in all components

### CODE REVIEW

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { useEffect, useState } from 'react';
 import styles from './admins.module.css';
 import Modal from './Modal/modal';
+import Buttons from '../Shared/Button';
 
 const Admins = () => {
   const [listAdmins, setListAdmin] = useState([]);
```
Comment (GinaSchP) [inline src/Components/Admins/index.js:7]:
```suggestion
  const [listAdmins, setListAdmins] = useState([]);
```

File: src/Components/Shared/Button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styles from './button.module.css';
+
+const Buttons = ({ type, action, variant, name }) => {
```
Comment (lmalvasia) [inline src/Components/Shared/Button/index.jsx:3]:
```suggestion
const Buttons = ({ type, onClick, variant, name }) => {
```

File: src/Components/Shared/Button/index.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  return (
+    <button type={type ? type : 'button'} className={`${styles[variant]}`} onClick={action}>
```
Comment (lmalvasia) [inline src/Components/Shared/Button/index.jsx:5]:
```suggestion
    <button type={type ? type : 'button'} className={`${styles[variant]}`} onClick={onClick}>
```

File: src/Components/Admins/index.js (hunk 1 above)
Comment (LucianoFranchino) [reply src/Components/Admins/index.js:7]: Resolved

File: src/Components/Shared/Button/index.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (LucianoFranchino) [reply src/Components/Shared/Button/index.jsx:3]: Resolved

File: src/Components/Shared/Button/index.jsx (hunk 3 above)
Comment (LucianoFranchino) [reply src/Components/Shared/Button/index.jsx:5]: Resolved

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**